Azure-Samples/iot-edge-opc-plc
# Deterministic Alarms and Conditions

## Purpose
* Enable deterministic alarms
Closes #45 

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code
- Run test suite (`DeterministicAlarmsTests`)
- Or run manually with `dotnet run --project src -- --dalm=./tests/DeterministicAlarmsTests/dalm001.json` and verify alarms and conditions in UAExpert

## What to Check
Verify that the following are valid
* Alarms and conditions occur as described in `dalm001.json` scenario

## Other Information
